openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -15,542 +15,657 @@ Status :
+|l10n_in                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_in_hr_payroll                      |                                   |
++----------------------------------------+-----------------------------------+
+| |new| l10n_in_schedule6                |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_it                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_jp                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_lu                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_ma                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_multilang                          |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_mx                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_nl                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_no                                 |                                   |
++----------------------------------------+-----------------------------------+
+| |new| l10n_nz                          |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_pa                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_pe                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_pl                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_pt                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_ro                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_sa                                 |                                   |
++----------------------------------------+-----------------------------------+
+|l10n_sg                                 |                               
```

Comment:
Equivalent module and renamed in OU in https://github.com/OCA/account-analytic/tree/9.0/sale_analytic_distribution